.changeset/pretty-goats-matter.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Ops, I think you didn't see [this comment](https://github.com/FuelLabs/fuels-ts/pull/1610#discussion_r1442762417).

See if it makes sense.